style/water.mss

```diff
@@ -33,6 +30,54 @@
       [way_pixels >= 64] { polygon-pattern-gamma: 0.6; }
     }
   }
+
+  [natural = 'water']::natural,
+  [landuse = 'reservoir']::landuse,
+  [waterway = 'riverbank']::waterway {
+    [zoom >= 0][zoom < 1][way_pixels >= 4],
+    [zoom >= 1][zoom < 2][way_pixels >= 16],
+    [zoom >= 2][zoom < 8][way_pixels >= 32],
+    [zoom >= 8] {
```

Comment:
Lets not reintroduce that - which we removed in #4060.

Reply:
Thank you for catching that. This is the problem with keeping a PR open so long.